core/domain/story_services.py

```diff
@@ -241,6 +246,226 @@ def save_new_story(committer_id, story):
         }])
 
 
+# Repository SAVE and DELETE methods.
+def apply_change_list(story_id, change_list):
+    """Applies a changelist to a story and returns the result.
+
+    Args:
+        story_id: str. ID of the given story.
+        change_list: list(dict). A change list to be applied to the given
+            story. Each entry in change_list is a dict that represents a
+            StoryChange.
+    object.
+
+    Returns:
+      Story. The resulting story domain object.
```

Comment:
indent by 4 not 2

/cc @apb7 FYI -- linter seems to pass, but it shouldn't. Could you please address this, or file an issue to keep track?

Reply:
Done.

Reply:
@seanlip: The docstring params linter uses regexp and checks for missing params, missing param type, differing param type, missing returns, missing return type, differing return type, missing yield, missing raises, missing raises type and differing raises type. I do not think it checks for the indentation of docstrings. We might require another check for it. I will see this. Thanks!

Reply:
I have added this to Issue #3905 under the Python checks.